### PR TITLE
Fixed Issue#5: Create component for single comment

### PR DESCRIPTION
**Pull request by kartikcode**
Added options for reply, like and dislike. Not sure about about the initial states for the counts for reacts. As of now, set it as zero.

### CODE REVIEW

**Comment by kartikcode** [comment]
comment

File: src/components/comment/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import ThumbUpAltIcon from "@material-ui/icons/ThumbUpAlt";
+import ThumbDownIcon from "@material-ui/icons/ThumbDown";
+import AccountCircleTwoToneIcon from "@material-ui/icons/AccountCircleTwoTone";
+import Box from "@material-ui/core/Box";
+
+function Comment(props) {
+  const [tds, setTds] = useState("inherit");
+  const [tus, setTus] = useState("inherit");
+  const [tuc, setTuc] = useState(0);
+  const [tdc, setTdc] = useState(0);
```
Comment (yashsriv) [inline src/components/comment/index.js:14]: Maybe have better variable names so people can understand what they represent.

File: src/components/comment/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+        aliquip
+      </div>
+      <div className="menu">
+        <Box m={0.5} p={0.8}>
+          <Button
+            variant="contained"
+            color="primary"
+            size="small"
+            onClick={handleClick()}
```
Comment (yashsriv) [inline src/components/comment/index.js:40]: the onClick handler must be passed a function. You actually called the function before passing so this won't work as you expect it to.

```suggestion
            onClick={handleClick}
```

File: src/components/comment/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useState } from "react";
```
Comment (NilayMajorwar) [inline src/components/comment/index.js:2]: Little nitpick: you can club this into a single line: `import React, {useState} from "react";`

**Pull request closed without merging**